packages/db/src/schema/companies.ts

```diff
@@ -32,6 +45,14 @@ export const CreateCompanySchema = createInsertSchema(Company, {
   description: z.string().optional(),
   industry: z.nativeEnum(Industry),
   location: z.string(),
-}) satisfies z.ZodType<
-  Omit<typeof Company.$inferInsert, "id" | "createdAt" | "updatedAt">
->;
+  averageHourlyPay: z.number(),
```

Comment:
Ok so decimal in drizzle still means string lmao, so we have the zod schema would have to just require them to be strings. See [this](https://stackoverflow.com/questions/76876763/drizzle-orm-decimal-mysql-is-a-string)

So you will have to change the create_Schema for both role and company to have z.string() for whatever we mark as decimal